### PR TITLE
Packages, consumer

### PR DESCRIPTION
**Pull request by l3r8yJ**
@h1alexbel take a look, please 

here are some changes for packages and I tried to create a `Consumer` interface. I'm not sure about `Consumer`, what do you think?

### CODE REVIEW

File: src/main/java/org/eocqrs/kafka/Consumer.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+package org.eocqrs.kafka;
+
+/**
+ * @author Ivan Ivanchuck (l3r8y@duck.com)
+ * @since 0.0.0
+ */
+public interface Consumer<X> {
+
+  void consume(int partition, Data<X> message);
```
Comment (h1alexbel) [inline src/main/java/org/eocqrs/kafka/Consumer.java:31]: I believe that this method should return data. But which type of the data? Maybe for now we can go with simple puzzle

Comment (h1alexbel) [reply src/main/java/org/eocqrs/kafka/Consumer.java:31]: I have some ideas: we can return Dataized<X> and then turn it in the real data fetched by the consumer

Comment (h1alexbel) [reply src/main/java/org/eocqrs/kafka/Consumer.java:31]: I mean turning object into some data

Comment (l3r8yJ) [reply src/main/java/org/eocqrs/kafka/Consumer.java]: @h1alexbel take a look, please at [this](https://github.com/eo-cqrs/eo-kafka/pull/48/commits/448b3c771ea6a9805e5d29cfb780cb24f88cfc77)

File: src/main/java/org/eocqrs/kafka/Consumer.java [hunk 2]
```diff
@@ -28,6 +28,6 @@
  */
 public interface Consumer<X> {
 
-  void consume(int partition, Data<X> message);
+  Dataized<?> consume(int partition, Data<X> message);
```
Comment (h1alexbel) [inline src/main/java/org/eocqrs/kafka/Consumer.java:31]: 1. To consume data we need partition and other meta information
2. I would name this method not consume, but dataized[read [this](https://www.yegor256.com/2018/08/22/builders-and-manipulators.html)]
3. finally, maybe we need a puzzle

what do you think @l3r8yJ

**Comment by l3r8yJ** [comment]
@h1alexbel take a look, please

**Review by h1alexbel — APPROVED** [review]

**Comment by 0pdd** [comment]
@l3r8yJ the puzzle [#49](https://github.com/eo-cqrs/eo-kafka/issues/49) is still not solved.

**Comment by h1alexbel** [comment]
@l3r8yJ thanks for contribution !

**Comment by 0pdd** [comment]
@l3r8yJ the puzzle [#54](https://github.com/eo-cqrs/eo-kafka/issues/54) is still not solved; solved: [#49](https://github.com/eo-cqrs/eo-kafka/issues/49).

**Comment by 0pdd** [comment]
@l3r8yJ 5 puzzles [#294](https://github.com/eo-cqrs/eo-kafka/issues/294), [#295](https://github.com/eo-cqrs/eo-kafka/issues/295), [#296](https://github.com/eo-cqrs/eo-kafka/issues/296), [#297](https://github.com/eo-cqrs/eo-kafka/issues/297), [#298](https://github.com/eo-cqrs/eo-kafka/issues/298) are still not solved; solved: [#49](https://github.com/eo-cqrs/eo-kafka/issues/49), [#54](https://github.com/eo-cqrs/eo-kafka/issues/54).

**Comment by 0pdd** [comment]
@l3r8yJ 4 puzzles [#294](https://github.com/eo-cqrs/eo-kafka/issues/294), [#295](https://github.com/eo-cqrs/eo-kafka/issues/295), [#297](https://github.com/eo-cqrs/eo-kafka/issues/297), [#unknown]() are still not solved; solved: [#296](https://github.com/eo-cqrs/eo-kafka/issues/296), [#298](https://github.com/eo-cqrs/eo-kafka/issues/298), [#49](https://github.com/eo-cqrs/eo-kafka/issues/49), [#54](https://github.com/eo-cqrs/eo-kafka/issues/54).

**Comment by 0pdd** [comment]
@l3r8yJ 2 puzzles [#297](https://github.com/eo-cqrs/eo-kafka/issues/297), [#unknown]() are still not solved; solved: [#294](https://github.com/eo-cqrs/eo-kafka/issues/294), [#295](https://github.com/eo-cqrs/eo-kafka/issues/295), [#296](https://github.com/eo-cqrs/eo-kafka/issues/296), [#298](https://github.com/eo-cqrs/eo-kafka/issues/298), [#49](https://github.com/eo-cqrs/eo-kafka/issues/49), [#54](https://github.com/eo-cqrs/eo-kafka/issues/54).

**Comment by 0pdd** [comment]
@l3r8yJ all 8 puzzles are solved here: [#294](https://github.com/eo-cqrs/eo-kafka/issues/294), [#295](https://github.com/eo-cqrs/eo-kafka/issues/295), [#296](https://github.com/eo-cqrs/eo-kafka/issues/296), [#297](https://github.com/eo-cqrs/eo-kafka/issues/297), [#298](https://github.com/eo-cqrs/eo-kafka/issues/298), [#49](https://github.com/eo-cqrs/eo-kafka/issues/49), [#54](https://github.com/eo-cqrs/eo-kafka/issues/54), [#unknown]().

**Pull request merged**